go.mod

```diff
@@ -53,6 +53,12 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require (
+	github.com/aws/aws-sdk-go-v2 v1.17.5
```

Comment:
Can you actually move this to the above `require` block.

Reply:
Will do.

Reply:
I moved this up to the earlier require block (not sure how this extra one got added in the first place). But I also moved `github.com/aws/aws-sdk-go` to the lower require block since it is only an indirect dependency.